src/interaction/interactiveTarget.js

```diff
@@ -42,5 +49,11 @@ module.exports = {
      * @todo Needs docs.
      * @private
      */
-    _touchDown: false
+    _touchDown: false,
+    
+    /**
+     * Used to track the global start position of a touchdown event on this target.
+     * @private
+     */
+    _touchDownPoint: new core.Point()
```

Comment:
Git it a shot ;)

My suggestion was based on the current implementation, but your new idea would require a bit more refactoring because currently the InteractionManager is trying to abstract the difference in touch and mouse away from the end API. I don't necessarily agree with this, so curious what you may have in mind.